@ecomplus/storefront-components/src/html/TheCart.html

```diff
@@ -0,0 +1,128 @@
+<div class="cart">
+    <transition-group enter-active-class="animated fadeInDown">
+      <div class="row" v-if="cart.items.length" key="list">
```

Comment:
```suggestion
      <div
        v-if="cart.items.length"
        class="row"
        key="list"
      >
```

Durante esta refatoração a intenção é fixar code style também, aqui por exemplo os atributos do elemento deveriam ser quebrados...